js/explore.js

```diff
@@ -0,0 +1,19 @@
+//Create the ability to input a word or sentence then output it in reverse. 
+// Do this by reading the word from the last character. Reading the input in reverse, 
+//take each letter and save into a new STRING, forming a new word. 
+//Printing this new word will show the initial word in reverse.
+
+function Reverse(string){
```

Comment:
caps `Reverse` should be downcase for a function `reverse`

don't name a function with caps.  Check jS syntax guidle lines if you have any questions